tests/pytests/test_misc.py

```diff
@@ -121,16 +121,6 @@ def test_deprecated_dcft_calls():
     assert err_substr in str(e.value)
 
 
-def test_deprecated_component_dipole():
-
-    #with pytest.warns(FutureWarning) as e:
-    psi4.set_variable("current dipole x", 5)
-
-    with pytest.warns(FutureWarning) as e:
-        ans = psi4.variable("current dipole x")
-
-    assert ans == 5
-
```

Comment:
I think you might be able to replace the warning with 

```
raise UpgradeHelper(key.upper(), key.upper()[:-2], 1.6, " Note the Debye -> a.u. units change.")
```

for https://github.com/psi4/psi4/blob/master/psi4/driver/p4util/python_helpers.py#L654 and the quadrupole that follows. That only fires for accessing, not setting. Then the lower part of the test above can remain with a `pytest.raises` instead of `pytest.warns`. Let me know if I've reasoned through wrong.